db/migrate/20210203160529_add_qualification_public_id_sequence.rb

```diff
@@ -0,0 +1,10 @@
+class AddQualificationPublicIdSequence < ActiveRecord::Migration[6.0]
+  def up
+    max_qualification_id = ApplicationQualification.maximum(:id)
+    create_sequence :qualifications_public_id_seq, start: max_qualification_id * 2
```

Comment:
Starting the sequence high enough to avoid duplicating and qualification IDs from the db. Twice the current max value seemed safe